### DISCUSSION TITLE
"An error has occurred. 0 Call to undefined function Astroid\getrusage() Call Stack

### DISCUSSION THREAD

**Discussion by AmirTalebi** [templaza/astroid-framework#865] [General]
Hello ,

After installing the quickstart package on the site and Joomla admin, I received the following error:

![chrome_tHiC9VYreU](https://github.com/user-attachments/assets/3687875f-798d-4b06-9ba6-44d521626052)
![chrome_yw3XbAFC1S](https://github.com/user-attachments/assets/386c42f0-e0f4-4b66-8ce7-bb9095dfe507)

Function Location
1 () JROOT/libraries/astroid/framework/library/astroid/Debugger.php:39"

Thank you

**Comment by Banuk1616** [comment]
Have you tried to install the Quickstart into an existing Joomla? The Quickstart is a complete package including Joomla and cannot be installed into an existing Joomla.

**Comment by AmirTalebi** [comment]
Dear Banuk1616 ,

The issue has been resolved

Thank you.
